utility/src/main/java/org/oppia/util/firebase/CrashlyticsWrapper.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for providing custom crash reporting to Firebase Crashlytics */
+@Singleton
+open class CrashlyticsWrapper @Inject constructor(
```

Comment:
We should generally discourage inheritance. Could we make this an interface, instead, and mock that? Then, we could provide an implementation of the wrapper that actually associates with the FirebaseCrashlytics implementation, and another that provides a fake implementation. You could use Mockito for the latter, but it might be cleaner to build an actual fake from which you can retrieve the logged exceptions & messages.

Reply:
Done, PTAL